static/sass/_snapcraft_market.scss

```diff
@@ -68,7 +68,8 @@
     }
 
     &:hover {
-      border-color: $color-link;
+      outline: $bar-thickness solid $color-focus;
```

Comment:
It's for UX/design to decide, but we probably shouldn't use the same styles for hover and for focus. Focus is much stronger state, only one element on page should have it.

Adding this style as hover means one element may have focus and this may appear focused just by hovering.